src/mmrelay/matrix_utils.py

```
@@ -1127,10 +1217,32 @@
             return False
         except Exception as e:
             # Handle other exceptions during login (e.g., network errors)
-            logger.error(f"Login exception: {e}")
-            logger.error(f"Exception type: {type(e)}")
+            error_type = type(e).__name__
+            logger.error(f"Login failed with {error_type}: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)